Makefile

```
@@ -211,6 +211,12 @@ check.docs:
 	@$(MAKE) docs
 	@tests/scripts/validate_modified_files.sh docs
 
+check.codegen:
+	@$(MAKE) codegen
+	@tests/scripts/validate_modified_files.sh codegen
+check.crds:
+	@$(MAKE) crds
+	@tests/scripts/validate_modified_files.sh crd
```

Ditto here. Why run make again instead of adding `crds` as a dependency of the target?

```suggestion
check.crds: crds
	@tests/scripts/validate_modified_files.sh crd
```